File: Programming Basics with Python/For-цикъл/salary.py

sites_count = int(input())
salary = int(input())

fb_count = 0
ig_count = 0
rd_count = 0

for i in range(1, sites_count+1):
    site = input()
    if site == "Facebook":
        fb_count += 1
    elif site == "Instagram":
        ig_count += 1
    elif site == "Reddit":
        rd_count += 1

salary = salary - ((fb_count * 150)+(ig_count * 100)+(rd_count * 50))

if salary > 0:
    print(salary)
else:
    print("You have lost your salary.")

File: Programming Basics with Java/For-loop/Salary.java
import java.util.Scanner;

public class Salary 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        int tabs = Integer.parseInt(input.nextLine());
        int salary = Integer.parseInt(input.nextLine());

        for (int i = 0; i < tabs; i++)
        {
            if (salary <= 0)
            {
                System.out.println("You have lost your salary.");
                break;
            }

            String text = input.nextLine();

            if (text.equals("Facebook"))
            {
                salary -= 150;
            }
            else if (text.equals("Instagram"))
            {
                salary -= 100;
            }
            else if (text.equals("Reddit"))
            {
                salary -= 50;
            }
        }

        if (salary > 0)
        {
            System.out.println(salary);
        }
    }
}
